src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -20,13 +20,16 @@ import ExposureNotification
 import UIKit
 
 public enum ENATaskIdentifier: String, CaseIterable {
-	case exposureNotification = "exposure-notification"
+	// only one task identifier is allowed have the .exposure-notification suffix
+	case detectExposures = "detect-exposures.exposure-notification"
 	case fetchTestResults = "fetch-test-results"
 
 	var backgroundTaskScheduleInterval: TimeInterval {
 		switch self {
-		case .exposureNotification: return 15 * 60 // 2 * 60 * 60 // set to trigger every 2 hours
-		case .fetchTestResults: return 5 * 60 // 30 * 60     // set to trigger every 30 min
+		// set to trigger every 2 hours
+		case .detectExposures: return 2 * 60 * 60
+		// set to trigger every 30 min
+		case .fetchTestResults: return 2 * 60 * 60
```

Comment:
Comment says 30 minutes; but two hours a returned

Reply:
It's the comment that's wrong, will fix that now. Good catch!